src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -105,6 +107,10 @@ open class JicofoServices(
 
         val authenticationIqHandler = authenticationAuthority?.let { AuthenticationIqHandler(it) }
 
+        iqHandler = IqHandler(focusManager).apply {
+            focusManager.addXmppConnectionListener { init(it) }
```

Comment:
I know Intellij says to use a lambda in these cases since the interface only has a single method, but I find it pretty confusing in the cases where it's an interface that's being implemented.  I think writing it out in full would be a lot clearer (it's basically impossible to infer what `it` represents here, let alone that the handler there is for a specific `xmppConnectionInitialized` event).

I.e.:

```
focusManager.addXmppConnectionListener(object : XmppConnectionListener {
    override fun xmppConnectionInitialized(xmppConnection: XmppConnection) {
        init(xmppConnection)
    }
})
```